styles/stylequery.css

```diff
@@ -59,20 +53,203 @@
   }
 
-  .midle-section {
-    display: grid;
-    grid-template-columns: 0.3fr 1fr 1fr 1fr;
-    grid-template-rows: 1fr 1fr 1fr 1fr;
-    grid-template-areas:
-      "projects project1"
-      "project2 project3"
-      "project4 project5"
-      "project6";
+  .project1 {
+    margin: 0;
+    width: 50vw;
+    height: 330px;
+    background-size: cover;
+  }
+
+  .project2 {
+    margin-left: 0;
+  }
+
+  .project3 {
+    margin-right: 0;
+    width: 48vw;
+    height: 320px;
+    background-size: cover;
+  }
+
+  .project4 {
+    margin-left: 0;
+    width: 30vw;
+    height: 300px;
+    background-size: cover;
+  }
+
+  .proyect-foot1 {
+    display: none;
+  }
+
+  .proyect-foot2 {
+    display: none;
+  }
+
+  .proyect-foot3 {
+    display: none;
+  }
+
+  .proyect-foot4 {
+    display: none;
+  }
+
+  .proyect-foot5 {
+    display: none;
+  }
+
+  .proyect-foot6 {
+    display: none;
+  }
+
+  .picture6 {
+    grid-area: picture6;
+
+    /* display: block; */
+    background-image: url(/pict-mobile/Rectangle\ 21\ \(5\) .svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  .ornament1 {
+    grid-area: ornament1;
+    display: block;
+    background-image: url(/pic-desktop/image-geometry_5.svg), url(/pic-desktop/image-geometry_4.svg);
+    background-repeat: no-repeat no-repeat;
+    background-size: contain contain;
+    background-size: 124px 124px 87px 87px;
+    background-position: left top, right top;
+  }
+
+  .ornament2 {
+    grid-area: ornament2;
+    display: block;
+    background-image: url(/pic-desktop/Shape.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-size: 130px 130px;
+    background-position: left bottom;
+  }
+
+  .ornament3 {
+    grid-area: ornament3;
+    display: block;
+    background-image: url(/pic-desktop/image-geometry_2.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-size: 430px 280px;
+    background-position: right bottom;
+    padding: 200px 0 120px 300px;
+    margin: -200px 0 -120px -300px;
+  }
+
+  .ornament4 {
+    grid-area: ornament4;
+    display: block;
+    background-image: url(/pic-desktop/Group\ 83.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-size: 350px 175px;
+    background-position: left top;
+    padding: 0 350px 200px 0;
+    margin: 240px -350px -200px 180px;
+  }
+
+  .works-cards1 {
+    grid-area: works-cards1;
+  }
+
+  .works-cards1 .proyect-foot1 {
+    display: none;
+  }
+
+  .works-cards1:hover .proyect-foot1 {
+    display: block;
+  }
+
+  .works-cards2 {
+    grid-area: works-cards2;
+  }
+
+  .works-cards2 .proyect-foot2 {
+    display: none;
+  }
+
+  .works-cards2:hover .proyect-foot2 {
+    display: block;
+    width: 400px;
+  }
+
+  .works-cards3 {
+    grid-area: works-cards3;
+  }
+
+  .works-cards3 .proyect-foot3 {
+    display: none;
+  }
+
+  .works-cards3:hover .proyect-foot3 {
+    display: block;
+  }
```

Comment:
- [ ] One of the main objectives of this project is to understand how to parse a Figma design to create a UI. From the project requirement, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9&t=WwXca8i3mkeo5TD6-0) as much as possible. But comparing your design to the Figma design, you can see there are some differences
  - Apart from the first card, when each project image is hovered, you should have the project details on top of the project image. 

### YOUR DESIGN
![Screenshot from 2022-12-11 04-30-31](https://user-images.githubusercontent.com/92303891/206886654-cf47c539-45dd-4926-a523-76ee75fc5f30.png)

### FIGMA DESIGN
![Screenshot from 2022-12-11 04-37-57](https://user-images.githubusercontent.com/92303891/206886700-d39eb613-6c1d-4bd5-a429-e35e2ee439c9.png)